### DISCUSSION TITLE
The switch pro controller does not recognize it when I connect it via usb

### DISCUSSION THREAD

**Discussion by CharmanderX151** [o0Zz/sys-con#54] [Q&A]
good, I comment that my swith is in the latest version of the system as also has the latest version of sys-con. what happens is that when connecting the controller by usb does not recognize it but when connecting it inhalambrica form this recognizes it without problems.

I found a notepad in the sys - con folder that writes the records when connecting the controller and there are several errors.

log

|I|00:00:13.733|1A148330| -----------------------------------------------------
|I|00:00:13.774|1A148330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:00:13.793|1A148330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:00:13.973|1A148330| USB configuration: Discovery mode(0), Auto add controller(true)
|I|00:02:46.756|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:47.813|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:47.826|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:47.839|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:47.848|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:47.861|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:47.872|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:47.882|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:48.291|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:49.342|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:49.355|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:49.365|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:49.377|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:49.389|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:49.398|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:49.410|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:49.827|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:50.879|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:50.894|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:50.910|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:50.923|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:50.928|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:50.934|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:50.943|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:50.949|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:51.998|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:52.006|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:52.011|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:52.017|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:52.025|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:52.031|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:52.036|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:52.389|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:53.436|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:53.444|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:53.449|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:53.458|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:53.464|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:53.469|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:53.478|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:53.923|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:54.968|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:54.977|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:54.983|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:54.992|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:54.998|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:55.006|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:55.012|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:04:09.782|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:04:11.084|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:04:11.090|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|E|00:04:11.101|1A1410D0| SwitchUSBInterface[045e-0b12] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:04:11.107|1A1410D0| Controller[045e-0b12] Failed to open interface 1 !
|E|00:04:11.118|1A1410D0| Controller[045e-0b12] Failed to open interfaces !
|E|00:04:11.124|1A1410D0| Controller[045e-0b12] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:04:11.135|1A1410D0| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:04:12.852|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:04:14.156|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:04:14.171|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|I|00:04:14.188|1A1410D0| SwitchHDLHandler[045e-0b12] Initialized !
|I|00:04:14.199|1A1410D0| Controller[045e-0b12] plugged !
|I|00:04:17.651|1A141098| USBInterface state was changed !
|I|00:04:17.658|1A141098| Controller[045e-0b12] unplugged !
|I|00:04:17.670|1A141098| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:04:19.766|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:04:21.058|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:04:21.069|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|I|00:04:21.088|1A1410D0| SwitchHDLHandler[045e-0b12] Initialized !
|I|00:04:21.098|1A1410D0| Controller[045e-0b12] plugged !
|I|00:04:26.099|1A141098| USBInterface state was changed !
|I|00:04:26.106|1A141098| Controller[045e-0b12] unplugged !
|I|00:04:26.117|1A141098| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:04:29.236|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:04:30.534|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:04:30.541|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|I|00:04:30.565|1A1410D0| SwitchHDLHandler[045e-0b12] Initialized !
|I|00:04:30.571|1A1410D0| Controller[045e-0b12] plugged !
|I|00:05:11.923|1A141098| USBInterface state was changed !
|I|00:05:11.934|1A141098| Controller[045e-0b12] unplugged !
|I|00:05:11.941|1A141098| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:05:16.598|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:05:17.899|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:05:17.906|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|I|00:05:17.931|1A1410D0| SwitchHDLHandler[045e-0b12] Initialized !
|I|00:05:17.937|1A1410D0| Controller[045e-0b12] plugged !
|I|00:05:20.115|1A141098| USBInterface state was changed !
|I|00:05:20.122|1A141098| Controller[045e-0b12] unplugged !
|I|00:05:20.135|1A141098| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:05:24.018|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:05:25.325|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:05:25.337|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:05:25.349|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:05:25.356|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:05:25.368|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:05:25.374|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:05:25.386|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:05:25.555|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:05:26.864|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:05:26.872|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:05:26.883|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:05:26.890|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:05:26.901|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:05:26.908|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:05:26.920|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:05:27.090|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:05:28.405|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:05:28.417|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:05:28.424|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:05:28.438|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:05:28.448|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:05:28.455|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:05:28.467|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:05:28.631|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:05:29.936|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:05:29.947|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:05:29.955|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:05:29.965|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:05:29.971|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:05:29.982|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:05:29.989|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:05:30.165|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:05:31.469|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:05:31.481|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:05:31.488|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:05:31.498|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:05:31.505|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:05:31.516|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:05:31.540|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:06:54.235|1A1410D0| Trying to initialize USB device: [045e-0b12] (Class: 0xFF, SubClass: 0x47, Protocol: 0xD0, bcd: 0x0516)...
|I|00:06:55.276|1A1410D0| Controller successfully loaded (B=1, A=2, Y=3, X=4, ...) !
|I|00:06:55.287|1A1410D0| Initializing Xbox One controller (Interface count: 3) ...
|I|00:06:55.311|1A1410D0| SwitchHDLHandler[045e-0b12] Initialized !
|I|00:06:55.322|1A1410D0| Controller[045e-0b12] plugged !
|I|00:06:58.268|1A141098| USBInterface state was changed !
|I|00:06:58.277|1A141098| Controller[045e-0b12] unplugged !
|I|00:06:58.289|1A141098| SwitchHDLHandler[045e-0b12] Uninitialized !
|I|00:07:12.668|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:07:13.706|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:07:13.715|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:07:13.727|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:07:13.739|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:07:13.748|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:07:13.760|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:07:13.771|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:07:14.202|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:07:15.243|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:07:15.254|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:07:15.263|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:07:15.274|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:07:15.285|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:07:15.294|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:07:15.307|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:07:15.737|1A1410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:07:16.775|1A1410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:07:16.787|1A1410D0| Initializing Switch (Interface count: 1) ...
|E|00:07:16.796|1A1410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:07:16.808|1A1410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:07:16.820|1A1410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:07:16.828|1A1410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:07:16.841|1A1410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:08:00.574|40F48330| -----------------------------------------------------
|I|00:08:00.588|40F48330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:08:00.599|40F48330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:08:00.864|40F48330| USB configuration: Discovery mode(0), Auto add controller(true)
|I|00:08:09.723|0E948330| -----------------------------------------------------
|I|00:08:09.737|0E948330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:08:09.750|0E948330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:08:10.003|0E948330| USB configuration: Discovery mode(0), Auto add controller(true)
|I|00:01:43.740|7E748330| -----------------------------------------------------
|I|00:01:43.754|7E748330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:01:43.767|7E748330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:01:44.038|7E748330| USB configuration: Discovery mode(0), Auto add controller(true)
|I|00:01:45.267|54748330| -----------------------------------------------------
|I|00:01:45.283|54748330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:01:45.296|54748330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:01:45.572|54748330| USB configuration: Discovery mode(0), Auto add controller(true)
|I|00:01:52.065|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:01:53.134|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:01:53.146|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:01:53.155|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:01:53.167|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:01:53.176|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:01:53.188|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:01:53.200|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:53.601|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:01:54.869|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:01:54.883|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:01:54.893|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:01:54.906|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:01:54.920|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:01:54.931|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:01:54.940|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:55.137|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:01:56.388|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:01:56.401|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:01:56.413|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:01:56.423|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:01:56.435|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:01:56.448|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:01:56.456|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:56.673|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:01:57.889|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:01:57.902|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:01:57.915|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:01:57.923|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:01:57.935|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:01:57.948|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:01:57.956|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:58.215|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:01:59.425|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:01:59.439|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:01:59.452|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:01:59.464|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:01:59.473|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:01:59.486|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:01:59.498|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:59.748|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:00.967|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:00.985|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:01.004|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:01.018|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:01.031|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:01.039|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:01.052|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:02:01.281|547410D0| Trying to initialize USB device: [057e-2009] (Class: 0x00, SubClass: 0x00, Protocol: 0x00, bcd: 0x0200)...
|I|00:02:02.497|547410D0| Controller successfully loaded (B=3, A=4, Y=1, X=2, ...) !
|I|00:02:02.506|547410D0| Initializing Switch (Interface count: 1) ...
|E|00:02:02.518|547410D0| SwitchUSBInterface[057e-2009] Failed to acquire USB interface - Error: 0x25A8C (Module: 0x8C, Desc: 0x12D) !
|E|00:02:02.531|547410D0| Controller[057e-2009] Failed to open interface 1 !
|E|00:02:02.539|547410D0| Controller[057e-2009] Failed to open interfaces !
|E|00:02:02.552|547410D0| Controller[057e-2009] Failed to initialize controller: Error: 0x6F (Module: 0x6F, Desc: 0x0)
|I|00:02:02.565|547410D0| SwitchHDLHandler[057e-2009] Uninitialized !
|I|00:01:29.359|2DD48330| -----------------------------------------------------
|I|00:01:29.370|2DD48330| SYS-CON started 1.4.3+0-0682ed5 (Build date: Oct 19 2024 09:53:24)
|I|00:01:29.381|2DD48330| OS version: 19.0.1 (Built with Atmosphere 1.7.x-1.8.x-989fb7be)
|I|00:01:29.609|2DD48330| USB configuration: Discovery mode(0), Auto add controller(true)
[log.log](https://github.com/user-attachments/files/17968331/log.log)

**Comment by o0Zz** [comment]
Did you read troubleshooting guide ?
https://github.com/o0Zz/sys-con/blob/master/doc/Troubleshooting.md

```This error occur when two drivers try to acquire the controller. Most of the time, this issue occured when you plug an official switch controller and the switch itself try to open it.```

Is you pro controller paired with your Console ?

Try below steps:
 1) Disable sys-con 
 2) Plug Pro via USB, the console should automatically pair it
 3) Un-plug pro controller
 4) Enable sys-con.
 
 Verify if the problem is fixed.
 If it's not fixed,. you need to understand why to switch try to open the usb controller before sys-con. Or set discovery_mode=1 as state in the troubleshooting guide

**Reply by CharmanderX151** [reply]
good thanks for answering, disable the sys-con and incidentally the mission control and trying to connect the controller this still does not recognize it, but if I activate the mission control this without any problem detects my Xbox controller via usb, but I still can not use the switch pro controller.


I decided to use the latest update of sys-con in the switch of a friend and this without problems grabs the switch pro control.

His switch has an instinc chip and mine has a picofly chip, do you think it has something to do with it?

Finally if I saw your guide and forgive my ignorance but I'm not very good at reading code, if possible show me an example of where exactly I should change the value of , discovery_mode=0,
discovery_vidpid=
I would be very grateful for it.

**Reply by o0Zz** [reply]
Please follow this https://github.com/o0Zz/sys-con/blob/master/doc/Troubleshooting.md#i-got-error-failed-to-acquire-usb-interface---error-0x25a8c-

**Reply by CharmanderX151** [reply]
I have already followed the steps and it still does not work, what else could I try?

I attach a picture of what I edited in the code, so it is correct?
![SYS- CON EDIT](https://github.com/user-attachments/assets/a6670cbe-062e-493d-b0cb-51c781f18ec4)
